app/views/home/index.html.slim

```diff
@@ -2,14 +2,7 @@
   row
     .row
       .col-9
-        h2.justify-content-center.mt-4 Association of co-owners of an apartment building
-        p.mt-4
-          | Our site saves you time and reduces the number of routine tasks. Personal accounts of
-          | residents increase the transparency of work and trust of residents. It is intended
-          | for use by both the management of condominiums and residents of the house. If you are
-          | an energetic person or a group of people who want positive changes in the management
-          | of your house, or even the head of a long-established condominium - you are in the
-          | right place!
+        h2.justify-content-center.mt-4 #{t 'home.title'}
+        p.mt-4 #{t 'home.osbb_app_goal'}
       .col-3.mt-3.mb-2
-        .building-img
-          = image_tag 'building.jpg', class: 'rounded imd-build ml-5', alt: 'building.jpg don`t loaded'
+        .building-img = image_tag 'building.jpg', class: 'rounded imd-build ml-5', alt: "#{t 'home.alt_text'}"
```

Comment:
and here to add brackets if you can skip brackets it doesn't mean that you should do it

Reply:
done